### PR TITLE
fix #17

### LINKED ISSUE #17: Why unnecessary arguments?
``-encrypt``
Argument is not necessary as this project does only one thing.

``-file``
Is not necessary too. This project only encrypts files, not strings, its not necessary.

``-decrypt``
This can be detected using file suffix. Such as .enc or .encrypted. Not necessary too.

You're making this project much more complicated than it should be. Please, make it simple, like Go. :)

### PR DESCRIPTION
**Pull request by zachcheung**
* remove `-encrypt`, `-decrypt`, `-file` flags, detect automatically
* use `.enc` suffix instead of long `.encrypted`
* don't panic
* add `-noclipboard` option

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,27 +35,27 @@ brew tap mimoo/eureka && brew install eureka
 **1.** You are trying to send *Bob* the file `myfile.txt`.Start by encrypting the file via:
 
 ```
-eureka -encrypt -file myfile.txt
+eureka myfile.txt
 ```
 
-which will return a one-time 256-bit AES key and create a new `myfile.txt.encrypted` file:
+which will return a one-time 256-bit AES key and create a new `myfile.txt.enc` file:
```
Comment (mimoo) [inline README.md]: what's the motive behind shortening this? It's true that extensions tend to be small, but I'm thinking of someone who encrypted something a while ago and forgot about what this `.enc` file on his desktop is.

File: eureka.go [hunk 2]
```diff
@@ -105,18 +113,27 @@ func main() {
...
-	if (*encrypt == false && *decrypt == false) || *inFile == "" {
+	if len(flag.Args()) == 0 {
 		fmt.Println("===================ᕙ(⇀‸↼‶)ᕗ===================")
 		fmt.Println(" Eureka is a tool to help you encrypt/decrypt a file")
 		fmt.Println(" to encrypt:")
-		fmt.Println("     eureka -encrypt -file [your-file]")
+		fmt.Println("     eureka your-file")
 		fmt.Println(" to decrypt:")
-		fmt.Println("     eureka -decrypt -file [encrypted-file]")
+		fmt.Println("     eureka encrypted-file-end-with-dot-enc")
```
Comment (mimoo) [inline eureka.go]: that's not very clear to me : o why not just `your-file.encrypted`

**Review by mimoo — APPROVED** [review]
Awesome PR! These are some good usability improvements :)

Just left two comments about the approach that I disagree with, feel free to ignore or address and merge.

File: README.md (hunk 1 above)
Comment (zachcheung) [reply README.md]: borrowed from #17 comment.

File: eureka.go (hunk 2 above)
Comment (zachcheung) [reply eureka.go]: ha, you got me, should be `encrypted-file.enc` at that time

**Comment by mimoo** [comment]
not sure if you could merge yourself? Thanks again!

**Comment by zachcheung** [comment]
> not sure if you could merge yourself? Thanks again!

didn't pay attention about it. not big deal~

**Pull request merged**